src/redux/actions/userActions.js

```diff
@@ -55,19 +57,16 @@ export const registerUserAsync = (user, openedFrom, postingOwnerId) => {
   return async (dispatch) => {
     try {
       dispatch(isUserFetching());
-      const createUserResp = await axios.post(
-        "http://localhost:3001/api/users",
-        {
-          userName,
-          firstName,
-          lastName,
-          email,
-          postalCode,
-          dateRegistered,
-          password,
-          isGoogleUser,
-        }
-      );
+      const createUserResp = await axios.post(BASE_URL, {
```

Comment:
im assuming the base url is like http://localhost:3000/api so here you're missing the /users part of the path